음양-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(absolutes, signs) {
+    const finalArray = absolutes.map((item, index) =>
+        signs[index] ? item : item * -1
+    );
+
+    const sum = finalArray.reduce((a, b) => a + b);
+
+    return sum;
+}
```

Comment:
```suggestion
function solution(absolutes, signs) {
    return absolutes.reduce((acc, cur, idx) => signs[idx] ? acc + cur : acc - cur, 0)
}
```

map의 콜백함수의 로직을 reduce의 콜백함수안에 넣어주면 결과적으로 배열의 순회를 1회 줄일 수 있습니다.